feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -469,8 +471,8 @@ class BillingWrapper(
                 var type: ProductType?
                 var presentedOffering: String?
                 synchronized(this@BillingWrapper) {
-                    type = productTypes[purchase.sku]
-                    presentedOffering = presentedOfferingsByProductIdentifier[purchase.sku]
+                    type = productTypes[purchase.skus[0]]
```

Comment:
```suggestion
                    type = productTypes[purchase.skus.getOrNull(0)]
```

i know it's functionally the same, but i think it's a good reminder to the reader that it could be null

Reply:
There should be always one sku though, `sku` wasn't nullable before